src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,22 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "actors")
 public class Actor implements Cloneable {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @OneToOne(fetch = FetchType.EAGER)
     private Country country;
```

Comment:
so we should create new country for each actor? maybe better will be `ManyToOne`?

Reply:
yeah, i got it.